addons/calendar/static/tests/helpers/mock_server/models/mail_activity.js

```diff
@@ -15,16 +15,20 @@ patch(MockServer.prototype, {
      * @override
      */
     async _performRPC(route, args) {
-        if (args.model === 'mail.activity' && args.method === 'action_create_calendar_event') {
+        if (args.model === "mail.activity" && args.method === "action_create_calendar_event") {
             return {
-                type: 'ir.actions.act_window',
+                type: "ir.actions.act_window",
                 name: "Meetings",
-                res_model: 'calendar.event',
-                view_mode: 'calendar',
-                views: [[false, 'calendar']],
-                target: 'current',
+                res_model: "calendar.event",
+                view_mode: "calendar",
+                views: [[false, "calendar"]],
+                target: "current",
             };
         }
+        if (args.model === "mail.activity" && args.method === "unlink_w_meeting") {
```

Comment:
This method should also unlink the related `calendar_event_id`.
Maybe not necessary for the purpose of this test, but as it's very simple to add, better get it right immediately.